mod/wikirate/set/type_plus_right/campaign/participant.rb

```diff
@@ -0,0 +1,27 @@
+
+
+# this is pretending as a pointer card
+def item_names args={}
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `item_names(*)` if you want the method to accept any arguments but don't care about them.<br>Use def with parentheses when there are parameters.<br>Surrounding space missing in default value assignment.